odtp/cli/result.py

```diff
@@ -0,0 +1,131 @@
+"""
+This scripts contains odtp subcommands for 'results'
+"""
+import logging
+
+import typer
+from typing_extensions import Annotated
+import zipfile
+
+import odtp.mongodb.db as db
+from odtp.storage import s3Manager
+import odtp.helpers.environment as odtp_env
+
+app = typer.Typer()
+
+## Adding listing so we can have multiple flags
+from typing import List
+
+
+@app.command()
+def prepare(    
+    dt_name: str = typer.Option(
+            None, "--digital-twin-name", help="Specify the digital twin name"
+    ),
+    dt_id: str = typer.Option(
+        None, "--digital-twin-id", help="Specify the digital twin ID"
+    ),
+    component_tag: str = typer.Option(
+        None, "--component-tags", help="Specify the components-tag (component-name:version)"
+    ),
+    project_path: str = typer.Option(
+        ..., "--project-path", help="Specify the path for the result execution"
+    ),
+):  
+    try:
+        if dt_name is None and dt_id is None:
```

Comment:
**suggestion**: I think overall the PR iff fine for me. but I would suggest to put the logic into the db function rather then in the cli command. There you can raise an exception if something goes wrong such as if the S3 connection can not be established. This way you will also better know what happened, whereas if you have the logic in the cli the command will just stop at a certain point in the try except block. 

I think it is just best practice to not have the logic in the cli command but rather in function that you call from the cli. That way the logic can also be reused later by the dashboard for example.